packages/core/src/common/props.ts

```diff
@@ -94,6 +98,14 @@ export interface IControlledProps {
     value?: string;
 }
 
+export interface IControlledProps2 {
```

Comment:
sorry to revisit this -- why are you making this change to remove `onChange` from the interface? you're `Omit`ing those fields anyway in `IInputGroupProps2` below..?

Reply:
The point of IControlledProps (as far I understand it) is to define the correct types we support, which is only ```value: string``` on ```value``` and ```defaultValue``` props, however ```onChange``` is compatible without any other change, so we're defining unnecessary properties (and even not following the original typing which is ```React.ChangeEvent``` not ```React.FormEventHandler``` (even though ```React.ChangeEventHandler``` could be used).

So unless there's an specific reason so not use the native type, I don't see why it should be redefined.